Explorer/Assets/DCL/Interaction/PlayerOriginated/Tests/WritePointerEventResultsSystemShould.cs

```diff
@@ -28,7 +28,7 @@ public class WritePointerEventResultsSystemShould : UnitySystemTestBase<WritePoi
         public void SetUp()
         {
             ISceneData sceneData = Substitute.For<ISceneData>();
-            sceneData.Geometry.Returns(new ParcelMathHelper.SceneGeometry(Vector3.zero, new ParcelMathHelper.SceneCircumscribedPlanes()));
+            sceneData.Geometry.Returns(new ParcelMathHelper.SceneGeometry(Vector3.zero, new ParcelMathHelper.SceneCircumscribedPlanes(), 0.0f));
```

Comment:
0.0f could be replaced with a const "MaxHeight"

Reply:
It's a default value, like Vector3.zero. It has no meaning.